cyclonedx/model/component.py

```diff
@@ -391,6 +392,20 @@ def purl(self) -> Optional[PackageURL]:
     @purl.setter
     def purl(self, purl: Optional[PackageURL]) -> None:
         self._purl = purl
+        
+    @property
+    def cpe(self) -> Optional[str]:
+        """
+        Specifies a well-formed CPE name that conforms to the CPE 2.2 or 2.3 specification.
```

Comment:
:+1:

Reply:
this is exactly the text from the [Spec](https://github.com/CycloneDX/specification/blob/master/schema/bom-1.4.xsd)
could you add the docs reference from the spec, too? 

```text
See https://nvd.nist.gov/products/cpe
```

Reply:
was fixed